src/daemon/ca/events.rs

```diff
@@ -421,21 +448,30 @@ pub enum CaEvtDet {
     },
 
     // Being a parent Events
+    /// A child was added to this (parent) CA
     ChildAdded {
         child: ChildHandle,
         id_cert: IdCert,
         resources: ResourceSet,
     },
+
+    /// A certificate was issued to the child of this (parent) CA
     ChildCertificateIssued {
         child: ChildHandle,
         resource_class_name: ResourceClassName,
         ki: KeyIdentifier,
     },
+
+    /// A child key was revoked.
     ChildKeyRevoked {
         child: ChildHandle,
         resource_class_name: ResourceClassName,
         ki: KeyIdentifier,
     },
+
+    /// Child certificates (for potentially multiple children) were updated
+    /// under a CA resource class. I.e. child certificates were issued,
+    /// removed, or suspended.
```

Comment:
Again, and perhaps I'm just losing my way here, are we talking about suspension of child certificates or of child CAs and certificates are not actually suspended?